### PR TITLE
fixed vertical aligment of items in the table

### PR DESCRIPTION
**Pull request by Sergey1011010**
before:
image
after:
image

### CODE REVIEW

**Comment by Sergey1011010** [comment]
@karelhala, @doruskova, please take a look

File: packages/rule-components/src/RuleTable/index.scss [hunk 1]
```diff
@@ -4,11 +4,30 @@
...
 
 .rhcs-c-rules-table {
-    .ins-c-table__toolbar.ins-m-footer .pf-c-pagination.pf-m-footer {
-        margin-left: auto;
+  .ins-c-table__toolbar.ins-m-footer .pf-c-pagination.pf-m-footer {
+    margin-left: auto;
+  }
+
+  .pf-c-select__menu .pf-c-check__label .ins-battery svg {
+    height: 1.25rem;
```
Comment (karelhala) [inline packages/rule-components/src/RuleTable/index.scss]: You should use `--pf-global--FontSize--xl`

Comment (Sergey1011010) [reply packages/rule-components/src/RuleTable/index.scss]: fixed

**Comment by karelhala** [comment]
waiting for @doruskova ACK.

**Comment by doruskova** [comment]
@Sergey1011010 @karelhala it's ok :)

**Pull request merged**